phys2bids/viz.py

```diff
@@ -33,6 +36,15 @@ def plot_trigger(time, trigger, fileprefix, tr, thr, num_timepoints_expected,
     figsize: tuple
         Desired size of the figure (see `matplotlib`),
         Default is {FIGSIZE}
+    tr: float
+        Repetition time
+    thr:float
+        Threshold used to detect the number of triggers
+    num_timepoints_expected: int
+        Number of timepoints expected by the user
+    figsize: integer tuple or list
+        Size of the figure X*Y
+        Default is {FIGSIZE}
     dpi: int
         Desired DPI of the figure (see `matplotlib`),
```

Comment:
numpydocs has a dedicated "See Also" section - you can put matplotlib with a link there:
https://numpydoc.readthedocs.io/en/latest/format.html#sections (point 12)